lib/test.js

```diff
@@ -243,33 +243,37 @@ var Test = {
         }
       };
 
-      global.contract = function(name, tests) {
-        // TODO: What is this / where did it come from? Do we need it?
-        if (typeof opts == "function") {
-          tests = name;
-          name = "";
-        }
```

Comment:
Looked into this - it's from a [2015 PR](https://github.com/trufflesuite/truffle-core/commit/73bbe4d7e6a4cdebafe0001fe046ae0b17138613) that added `opts` to this function as part of  optionally forcing contract redeployment before every test. Reversion replaced this strategy I think.